zebra-scan/src/storage.rs

```diff
@@ -89,10 +93,20 @@ impl Storage {
     ///
     /// This method can block while reading database files, so it must be inside spawn_blocking()
     /// in async code.
-    pub fn sapling_keys(&self) -> HashMap<SaplingScanningKey, Height> {
+    pub fn sapling_keys_birthdays(&self) -> HashMap<SaplingScanningKey, Height> {
```

Comment:
I think this function can be removed.

Reply:
Removed in https://github.com/ZcashFoundation/zebra/pull/8080/commits/53dcb31882b12f954e8ee4a4fa742249f3dd1faa